### ISSUE TITLE
Command line flag for rubocop config file

### ISSUE THREAD

**Issue by kamen-hursev** [brigade/haml-lint#118]
Are you willing to accept another command line flag `--rubocop-config` that allows explicitly setting the rubocop config file? 

I know you accept `HAML_LINT_RUBOCOP_CONF` environment variable, but it is difficult(impossible?) to manage with SublimeLinter. Some more info [here](https://github.com/jeroenj/SublimeLinter-contrib-haml-lint/issues/2)

**Comment by sds** [comment]
Hey @kamen-hursev,

This issue has surfaced in other projects, and my understanding (which I've gathered from others, since I don't personally use Sublime) is that it is perfectly possible to use environment variables in Sublime linters.

For example, this was implemented for the `slim-lint` project in https://github.com/sds/slim-lint/pull/6 to support the [Sublime-Linter-slim-lint](https://github.com/elstgav/SublimeLinter-slim-lint) package. I don't have details for you as I didn't implement it.

If for some reason this is still truly difficult, we can explore adding a flag.
